client/src/components/Messages/Message.tsx

```diff
@@ -43,19 +44,26 @@ export default function Message({
     enabled: false,
   });
   const blinker = message?.submitting && isSubmitting;
-
+  const { conversationId } = useParams();
   // debugging
   // useEffect(() => {
   //   console.log('isSubmitting:', isSubmitting);
   //   console.log('unfinished:', unfinished);
   // }, [isSubmitting, unfinished]);
+  const autoScroll = useRecoilValue(store.autoScroll);
 
   useEffect(() => {
     if (blinker && scrollToBottom && !abortScroll) {
       scrollToBottom();
     }
   }, [isSubmitting, blinker, text, scrollToBottom]);
 
+  useEffect(() => {
+    if (scrollToBottom && autoScroll) {
```

Comment:
Even though it shouldn't render in this case, I would just add a return condition if conversationId === 'new', return. This was a smart way of grabbing the convoId! 

I'm curious what kind of behavior there is when using Search, too, since it's pulling from params. Please test this. A search will change the params

Lastly, after thinking about this, you might be better off adding this to the <Messages/> component where Messages and we could probably move all the scrollToBottom() logic in Messages.

In other words, im not sure how important `message?.submitting` is.. if you are unsure what to do here, I can work on it after we merge this.